src/Illuminate/Auth/Access/Gate.php

```diff
@@ -162,7 +164,7 @@ public function check($ability, $arguments = [])
      *
      * @param  string  $ability
      * @param  array  $arguments
-     * @return callable
+     * @return mixed
```

Comment:
Think of it like you would document the return with an interface. Except in a more weird, PHP magic way...